service/api/prettyprint.go

```diff
@@ -88,10 +92,10 @@ func (v *Variable) writeTo(buf io.Writer, top, newlines, includeType bool, inden
 			} else if data.Children[0].OnlyAddr {
 				fmt.Fprintf(buf, "0x%x", v.Children[0].Addr)
 			} else {
-				v.Children[0].writeTo(buf, false, newlines, false, indent)
+				v.Children[0].writeTo(buf, false, newlines, !includeType, indent)
```

Comment:
It seems odd to negate what was passed in. I think I understand the sentiment, that if we say "don't print type info", if the type happens to be an interface w/ children we automatically print the types of the children, right?

Maybe we should let the user control this, and allow the user to specify via the `print` command whether type information should be included? If types should be included, then if the type is an interface type we also print out the type of each underlying element.